Tests/GRPCTests/ClientTLSFailureTests.swift

```diff
@@ -135,6 +149,43 @@ class ClientTLSFailureTests: GRPCTestCase {
       XCTFail("Expected NIOSSLError.handshakeFailed(BoringSSL.sslError)")
     }
   }
+  
+  func testClientConnectionFailsWhenServerIsUnknownWithHTTPTargetWindowSize() throws {
```

Comment:
Could you remove the extra tests since we're not actually validating a change in behaviour here? I think I'm okay just deferring to NIO HTTP/2 for the tests.